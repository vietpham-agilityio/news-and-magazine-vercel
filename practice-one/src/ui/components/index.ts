// common
export * from '@/ui/components/common';

// shared
export * from '@/ui/components/Video';
export * from '@/ui/components/Brand';
export * from '@/ui/components/NavItem';
export * from '@/ui/components/Comment';
export * from '@/ui/components/Carousel';
export * from '@/ui/components/SearchBar';
export * from '@/ui/components/PaperPost';
export * from '@/ui/components/PaperVideo';
export * from '@/ui/components/BurgerMenu';
export * from '@/ui/components/Card/CardSingle';
export * from '@/ui/components/Skeleton/CardPost';
export * from '@/ui/components/Skeleton/DetailPost';
export * from '@/ui/components/Skeleton/CardVertical';
export { default as CardPost } from '@/ui/components/Card/CardPost';