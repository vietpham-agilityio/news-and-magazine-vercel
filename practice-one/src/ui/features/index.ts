export * from '@/ui/features/TagGroup';
export * from '@/ui/features/SidebarCard';
export * from '@/ui/features/RightSidebar';
export * from '@/ui/features/HashtagGroup';

export { default as PostsGrid } from '@/ui/features/PostsGrid';
export { default as PostDetail } from '@/ui/features/PostDetail';
export { default as PostsByAttribute } from '@/ui/features/PostsByAttribute';
