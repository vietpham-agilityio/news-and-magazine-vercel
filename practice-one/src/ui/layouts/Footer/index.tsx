// components
import Link from 'next/link';
import Image from 'next/image';
import { Button, TitleSection, Comment, Typography } from '@/ui/components';

// icons
import { Instagram, MailBox, Twitter } from '@/ui/components/Icons';
import { FontWeight } from '@/types';

export const Footer = () => (
  <footer className="footer w-full hidden items-center justify-center bg-white-100 mb-12.5 sm:flex lg:mb-11">
    <div className="flex-1 bg-white-90 py-10 pr-6 rounded-e-[50px] mr-11 lg:mr-0 lg:pr-10">
      <div className="flex justify-center lg:justify-end pr-4 lg:pr-0">
        {/* footer content */}
        <div className="pl-6 lg:pl-10">
          <section className="w-100">
            <div className="mb-5">
              <TitleSection title="Mega news" />
            </div>
            <p className="pr-8 lg:pr-12 h-40 text-xs text-justify font-normal text-black-75">
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              eiusmod tempor incididunt ut labore et dolore magna aliqua.
              Egestas purus viverra accumsan in nisl nisi. Arcu cursus vitae
              congue mauris rhoncus aenean vel elit scelerisque. In egestas erat
              imperdiet sed euismod nisi porta lorem mollis. Morbi tristique
              senectus et netus. Mattis pellentesque id nibh tortor id aliquet
              lectus proin
            </p>
          </section>
          {/* Newsletters */}
          <section className="w-100">
            <div className="mb-5">
              <TitleSection title="Newsletters" />
            </div>
            <label htmlFor="search" className="relative mr-6 block">
              <div className="absolute left-6 top-4"></div>
              <div className="absolute right-3 top-4">
                <MailBox />
              </div>
              <input
                type="text"
                name="search"
                id="search"
                className="h-12 text-p bg-white-100 placeholder-dark-100 rounded-xl pl-3 pr-14 outline-none w-full lg:w-100"
                placeholder="Write your email ..."
              />
            </label>
          </section>
        </div>
        {/* footer connect */}
        <div className="ml-4 lg:ml-16 2xl:ml-30">
          <section className="link-group">
            <div className="mb-5">
              <TitleSection title="Categories" />
            </div>
            <ul>
              <li className="text-p text-dark-100 mb-2 hover:text-secondary-100">
                <Link href="/category/sports">Sports</Link>
              </li>
              <li className="text-p text-dark-100 mb-2 hover:text-secondary-100">
                <Link href="/category/entertainment">Entertainment</Link>
              </li>
              <li className="text-p text-dark-100 mb-2 hover:text-secondary-100">
                <Link href="/category/science-and-environment">Science</Link>
              </li>
              <li className="text-p text-dark-100 mb-2 hover:text-secondary-100">
                <Link href="/category/health-and-wellness">Health Living</Link>
              </li>
              <li className="text-p text-dark-100 mb-2 hover:text-secondary-100">
                <Link href="/category/lifestyle">Lifestyle</Link>
              </li>
              <li className="text-p text-dark-100 mb-3 hover:text-secondary-100">
                <Link href="/category/technology">Technology</Link>
              </li>
            </ul>
          </section>
          <section className="social-group">
            <div className="mb-5">
              <TitleSection title="Social network" />
            </div>
            <div className="h-12 flex items-center">
              <Button
                isMarginBottom
                label="Instagram"
                textColor="text-white-100"
                bgColor="bg-gradient-to-bl from-sunSet-0 to-sunSet-100"
                leftIcon={<Instagram />}
              />
              <div className="w-10 h-10 ml-4 bg-gradient-to-bl from-blueLinear-0 to-blueLinear-100 flex items-center justify-center rounded-xl cursor-pointer">
                <Twitter />
              </div>
            </div>
          </section>
        </div>
      </div>
      <div className="flex mt-8 bg-black-90 rounded-xl justify-center lg:justify-end">
        <div className="flex justify-between py-4 pr-4">
          <Typography
            tag="h2"
            weight={FontWeight.Light}
            additionalClasses="text-dark-75 text-p leading-5 w-100 lg:mr-15 2xl:mr-30"
          >
            Privacy Policy | Terms & Conditions
          </Typography>
          <Typography
            tag="h2"
            weight={FontWeight.Light}
            additionalClasses="text-dark-75 text-p leading-5"
          >
            All Copyright (C) 2022 Reserved
          </Typography>
        </div>
      </div>
    </div>
    <div className="hidden flex-1 h-full py-10 lg:flex">
      {/* comment group */}
      <section className="h-full bg-white-100 mx-5 w-full 2xl:w-90">
        <div className="mb-5">
          <TitleSection title="New Comments" />
        </div>
        <div className="comment-wrapper">
          <div className="mb-2">
            <Comment
              userName="Ellsmartx"
              content="How Nice Does This Look 😍 I Feel It Should Be Delicious, Thank You"
            />
          </div>
          <div className="mb-2">
            <Comment
              userName="Cassia"
              content="Take A Rest, I'll Be Cheer Up You Again In 2 Next Game Go Go Go"
            />
          </div>
          <div className="mb-2">
            <Comment
              userName="Amanda"
              content="You Were Stunning Today, Jan! 💗 Great Match 👍🏽👍🏽"
            />
          </div>
          <div>
            <Comment
              userName="Denis Simonassi"
              content="It Was A Great Match Today Janzi! You Did Great😉🇩🇪"
            />
          </div>
        </div>
      </section>
      {/* Social grid */}
      <section className="w-90 h-full bg-white-100 hidden 2xl:block ">
        <div className="mb-5">
          <TitleSection title="Follow On Instagram" />
        </div>
        <div className="media-wrapper grid grid-cols-3 grid-rows-3 gap-5">
          <Image
            width={104}
            height={104}
            className="rounded-3xl"
            src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
            alt="media-image"
          />
          <Image
            width={104}
            height={104}
            className="rounded-3xl"
            src="https://images.unsplash.com/photo-1534294228306-bd54eb9a7ba8?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MjQzfHxwbGFuZXR8ZW58MHx8MHx8fDA%3D"
            alt="media-image"
          />
          <Image
            width={104}
            height={104}
            className="rounded-3xl"
            src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
            alt="media-image"
          />
          <Image
            width={104}
            height={104}
            className="rounded-3xl"
            src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
            alt="media-image"
          />
          <Image
            width={104}
            height={104}
            className="rounded-3xl"
            src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
            alt="media-image"
          />
          <Image
            width={104}
            height={104}
            className="rounded-3xl"
            src="https://images.unsplash.com/photo-1545156521-77bd85671d30?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8NHx8cGxhbmV0fGVufDB8fDB8fHww"
            alt="media-image"
          />
          <Image
            width={104}
            height={104}
            className="rounded-3xl"
            src="https://images.unsplash.com/photo-1620421680906-275860f61e27?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8NTh8fHBsYW5ldHxlbnwwfHwwfHx8MA%3D%3D"
            alt="media-image"
          />
          <Image
            width={104}
            height={104}
            className="rounded-3xl"
            src="https://images.unsplash.com/photo-1632395627760-72e6eca7f9c7?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MzF8fHBsYW5ldHxlbnwwfHwwfHx8MA%3D%3D"
            alt="media-image"
          />
          <Image
            width={104}
            height={104}
            className="rounded-3xl"
            src="https://images.unsplash.com/photo-1481819613568-3701cbc70156?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8M3x8cGxhbmV0fGVufDB8fDB8fHww"
            alt="media-image"
          />
        </div>
      </section>
    </div>
  </footer>
);
