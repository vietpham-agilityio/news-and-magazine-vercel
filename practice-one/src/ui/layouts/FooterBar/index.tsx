// constants
import { flexCenter } from '@/constants';

// components
import { Button } from '@/ui/components/common';

// icons
import { Home, SaveTag, User, ChevronsUp } from '@/ui/components/Icons';

export const FooterBar = () => (
  <footer className="footer w-full flex items-center justify-between bg-white-100 h-17.5 fixed bottom-0 px-7.5 py-4 rounded-t-2xl shadow-inner sm:hidden">
    <Button
      label="Home"
      textColor="text-white-100"
      bgColor="bg-gradient-to-bl from-sunSet-0 to-sunSet-100"
      leftIcon={<Home />}
    />
    <div className={`${flexCenter} w-10 h-10`}>
      <SaveTag />
    </div>
    <div className={`${flexCenter} w-10 h-10`}>
      <User />
    </div>
    <div className={`${flexCenter} w-10 h-10`}>
      <ChevronsUp />
    </div>
  </footer>
);
