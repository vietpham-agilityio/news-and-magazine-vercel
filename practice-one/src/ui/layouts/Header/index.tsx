// components
import {
  Menu,
  Brand,
  Avatar,
  NavItem,
  BoxIcon,
  SearchBar,
} from '@/ui/components';
import { CategoryBurgerMenu, CategoryMenu } from '@/ui/features';

// icons
import { ArrowBottom, SaveTag, Search } from '@/ui/components/Icons';

export const Header = () => (
  <header className="flex flex-col items-center justify-center bg-white-100 h-auto mt-11 mx-2 sm:mx-6 sm:mt-4 lg:mt-11 lg:flex-row lg:container lg:mx-auto">
    <div className="h-12 w-full flex items-center justify-between">
      <div className="flex items-center w-full sm:w-166 lg:flex-0 lg:justify-between">
        {/* Burger menu */}
        <div className="mr-7.5 lg:hidden">
          <CategoryBurgerMenu />
        </div>
        {/* Search in small screen */}
        <div className="flex flex-1 sm:hidden">
          <SearchBar
            name="search"
            type="text"
            placeholder="Search everything"
          />
        </div>
        {/* Brand */}
        <div className="hidden sm:block">
          <Brand brandName="MEGA.news" url="/" />
        </div>
        <nav className="hidden w-119 items-center justify-center lg:flex">
          <div className="mr-8">
            <CategoryMenu />
          </div>
          <div className="mr-8">
            <NavItem name="Pages" url="/pages" />
          </div>
          <div className="mr-8">
            <NavItem name="Contact Us" url="/contact" />
          </div>
          <NavItem name="About Us" url="/about" />
        </nav>
      </div>

      {/*Search and user*/}
      <div className="w-87.5 items-center justify-end hidden sm:flex lg:w-164">
        <label htmlFor="search" className="relative mr-6 hidden xl:block">
          <div className="absolute right-4 top-3 cursor-pointer">
            <Search />
          </div>
          <input
            type="text"
            name="search"
            id="search"
            className="h-12 bg-white-90 placeholder-dark-100 rounded-xl outline-none pl-4 pr-14 lg:w-100"
            placeholder="Search Anything"
          />
        </label>
        <div className="flex xl:flex-1 justify-between">
          <div className="flex items-center mr-8 cursor-pointer xl:mr-0">
            <Avatar
              width={48}
              height={48}
              url="https://images.unsplash.com/photo-1535713875002-d1d0cf377fde?q=80&w=2680&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
              alt="Avatar"
            />
            <p className="text-sm font-medium text-dark-100 mx-2">Behzad</p>
            <ArrowBottom />
          </div>
          <BoxIcon icon={<SaveTag />} additionalClasses="w-12" />
        </div>
      </div>
    </div>

    {/*search on small screen*/}
    <div className="w-full h-14 hidden my-5 sm:flex lg:hidden">
    <SearchBar
        name="search"
        type="text"
        placeholder="Search everything"
      />
    </div>
  </header>
);
