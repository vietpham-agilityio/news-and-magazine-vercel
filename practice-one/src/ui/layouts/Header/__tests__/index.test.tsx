import { render } from '@testing-library/react';

// component
import { Header } from '@/ui/layouts';

let renderHeader: any;

describe('Header feature component', () => {
  beforeEach(() => {
    renderHeader = render(
      <Header />
    );
  });

  afterEach(() => {
    renderHeader.unmount();
  });

  it('Header should render macth snapshot', () => {
    expect(renderHeader.asFragment()).toMatchSnapshot();
  });
});
