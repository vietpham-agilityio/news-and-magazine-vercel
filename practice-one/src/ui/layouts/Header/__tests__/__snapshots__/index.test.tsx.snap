// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header feature component Header should render macth snapshot 1`] = `
<DocumentFragment>
  <header
    class="header container flex flex-col items-center justify-center bg-white-100 h-auto mx-auto mt-11 sm:mt-4 lg:mt-11 lg:flex-row"
  >
    <div
      class="h-12 w-full flex items-center justify-between"
    >
      <div
        class="flex items-center w-full sm:w-166 lg:flex-0 lg:justify-between"
      >
        <div
          class="mr-7.5 lg:hidden"
        >
          <div>
            <button
              aria-label="box-button"
              class="w-12 h-12 bg-white-90 flex items-center justify-center rounded-xl w-12"
            >
              <svg
                fill="none"
                height="16"
                width="18"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M.25 1.75C.25 1.086.797.5 1.5.5h15c.664 0 1.25.586 1.25 1.25 0 .703-.586 1.25-1.25 1.25h-15C.797 3 .25 2.453.25 1.75ZM.25 8c0-.664.547-1.25 1.25-1.25h15c.664 0 1.25.586 1.25 1.25 0 .703-.586 1.25-1.25 1.25h-15C.797 9.25.25 8.703.25 8Zm16.25 7.5h-15c-.703 0-1.25-.547-1.25-1.25C.25 13.586.797 13 1.5 13h15c.664 0 1.25.586 1.25 1.25 0 .703-.586 1.25-1.25 1.25Z"
                  fill="#3E3232"
                />
              </svg>
            </button>
          </div>
        </div>
        <div
          class="flex flex-1 sm:hidden"
        >
          <label
            class="relative flex-1"
            for="search"
          >
            <div
              class="absolute left-6 top-4"
            >
              <svg
                data-testid="three-dots-icon"
                fill="none"
                height="18"
                width="6"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3 13.063c1.172 0 2.188 1.015 2.188 2.187 0 1.21-1.016 2.188-2.188 2.188A2.184 2.184 0 0 1 .812 15.25c0-1.172.977-2.188 2.188-2.188Zm0-6.25c1.172 0 2.188 1.015 2.188 2.187 0 1.21-1.016 2.188-2.188 2.188A2.184 2.184 0 0 1 .812 9c0-1.172.977-2.188 2.188-2.188Zm0-1.875A2.184 2.184 0 0 1 .812 2.75C.813 1.578 1.79.562 3 .562c1.172 0 2.188 1.016 2.188 2.188 0 1.21-1.016 2.188-2.188 2.188Z"
                  fill="#3E3232"
                  fill-opacity="0.75"
                />
              </svg>
            </div>
            <div
              class="absolute right-4 top-3"
            >
              <svg
                data-testid="search-icon"
                fill="none"
                height="22"
                width="22"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M20.531 18.344a1.63 1.63 0 0 1 0 2.226 1.63 1.63 0 0 1-2.226 0l-4.649-4.687c-1.601 1.055-3.554 1.601-5.664 1.328-3.594-.508-6.484-3.438-6.953-6.992-.664-5.313 3.828-9.805 9.14-9.14 3.555.468 6.485 3.359 6.993 6.952a8.111 8.111 0 0 1-1.328 5.625l4.687 4.688ZM4.086 9.125c0 2.773 2.226 5 5 5 2.734 0 5-2.227 5-5 0-2.734-2.266-5-5-5-2.774 0-5 2.266-5 5Z"
                  fill="#3E3232"
                />
              </svg>
            </div>
            <input
              class="w-full h-12 bg-white-90 placeholder-dark-100 rounded-xl outline-none px-14"
              id="search"
              name="search"
              placeholder="Search everything"
              type="text"
            />
          </label>
        </div>
        <div
          class="hidden sm:block"
        >
          <a
            href="/"
          >
            <h1
              class="text-lg font-bold text-secondary-100"
            >
              MEGA.news
            </h1>
          </a>
        </div>
        <nav
          class="hidden w-119 items-center justify-center lg:flex"
        >
          <div
            class="mr-8"
          >
            <div
              class="flex items-center relative hover:cursor-pointer"
              data-testid="menu-item"
            >
              <p
                class="text-sm font-medium text-dark-100 hover:text-primary-100 mr-2"
              >
                Categories
              </p>
              <svg
                fill="none"
                height="7"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M6 6.75a.866.866 0 0 1-.629-.246L.996 2.129a.843.843 0 0 1 0-1.23.843.843 0 0 1 1.23 0L6 4.644 9.746.898a.843.843 0 0 1 1.23 0 .843.843 0 0 1 0 1.23L6.603 6.505A.848.848 0 0 1 6 6.75Z"
                  fill="#3E3232"
                  fill-opacity="0.5"
                />
              </svg>
            </div>
          </div>
          <div
            class="mr-8"
          >
            <a
              class="flex items-center relative"
              href="/pages"
            >
              <p
                class="text-sm font-medium text-dark-100 hover:text-primary-100"
              >
                Pages
              </p>
            </a>
          </div>
          <div
            class="mr-8"
          >
            <a
              class="flex items-center relative"
              href="/contact"
            >
              <p
                class="text-sm font-medium text-dark-100 hover:text-primary-100"
              >
                Contact Us
              </p>
            </a>
          </div>
          <a
            class="flex items-center relative"
            href="/about"
          >
            <p
              class="text-sm font-medium text-dark-100 hover:text-primary-100"
            >
              About Us
            </p>
          </a>
        </nav>
      </div>
      <div
        class="w-87.5 items-center justify-end hidden sm:flex lg:w-164"
      >
        <label
          class="relative mr-6 hidden xl:block"
          for="search"
        >
          <div
            class="absolute left-6 top-4"
          >
            <svg
              data-testid="three-dots-icon"
              fill="none"
              height="18"
              width="6"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3 13.063c1.172 0 2.188 1.015 2.188 2.187 0 1.21-1.016 2.188-2.188 2.188A2.184 2.184 0 0 1 .812 15.25c0-1.172.977-2.188 2.188-2.188Zm0-6.25c1.172 0 2.188 1.015 2.188 2.187 0 1.21-1.016 2.188-2.188 2.188A2.184 2.184 0 0 1 .812 9c0-1.172.977-2.188 2.188-2.188Zm0-1.875A2.184 2.184 0 0 1 .812 2.75C.813 1.578 1.79.562 3 .562c1.172 0 2.188 1.016 2.188 2.188 0 1.21-1.016 2.188-2.188 2.188Z"
                fill="#3E3232"
                fill-opacity="0.75"
              />
            </svg>
          </div>
          <div
            class="absolute right-4 top-3"
          >
            <svg
              data-testid="search-icon"
              fill="none"
              height="22"
              width="22"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M20.531 18.344a1.63 1.63 0 0 1 0 2.226 1.63 1.63 0 0 1-2.226 0l-4.649-4.687c-1.601 1.055-3.554 1.601-5.664 1.328-3.594-.508-6.484-3.438-6.953-6.992-.664-5.313 3.828-9.805 9.14-9.14 3.555.468 6.485 3.359 6.993 6.952a8.111 8.111 0 0 1-1.328 5.625l4.687 4.688ZM4.086 9.125c0 2.773 2.226 5 5 5 2.734 0 5-2.227 5-5 0-2.734-2.266-5-5-5-2.774 0-5 2.266-5 5Z"
                fill="#3E3232"
              />
            </svg>
          </div>
          <input
            class="h-12 bg-white-90 placeholder-dark-100 rounded-xl outline-none px-14 lg:w-100"
            id="search"
            name="search"
            placeholder="Search Anything"
            type="text"
          />
        </label>
        <div
          class="flex xl:flex-1 justify-between"
        >
          <div
            class="flex items-center mr-8 xl:mr-0"
          >
            <div
              style="width: 48px; height: 48px; position: relative;"
            >
              <img
                alt="Avatar"
                class="rounded-xl"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                sizes="100vw"
                src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1535713875002-d1d0cf377fde%3Fq%3D80%26w%3D2680%26auto%3Dformat%26fit%3Dcrop%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D&w=3840&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1535713875002-d1d0cf377fde%3Fq%3D80%26w%3D2680%26auto%3Dformat%26fit%3Dcrop%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1535713875002-d1d0cf377fde%3Fq%3D80%26w%3D2680%26auto%3Dformat%26fit%3Dcrop%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1535713875002-d1d0cf377fde%3Fq%3D80%26w%3D2680%26auto%3Dformat%26fit%3Dcrop%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1535713875002-d1d0cf377fde%3Fq%3D80%26w%3D2680%26auto%3Dformat%26fit%3Dcrop%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1535713875002-d1d0cf377fde%3Fq%3D80%26w%3D2680%26auto%3Dformat%26fit%3Dcrop%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1535713875002-d1d0cf377fde%3Fq%3D80%26w%3D2680%26auto%3Dformat%26fit%3Dcrop%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1535713875002-d1d0cf377fde%3Fq%3D80%26w%3D2680%26auto%3Dformat%26fit%3Dcrop%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1535713875002-d1d0cf377fde%3Fq%3D80%26w%3D2680%26auto%3Dformat%26fit%3Dcrop%26ixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D&w=3840&q=75 3840w"
                style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
              />
            </div>
            <p
              class="text-sm font-medium text-dark-100 mx-2"
            >
              Behzad
            </p>
            <svg
              fill="none"
              height="7"
              width="12"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M6 6.75a.866.866 0 0 1-.629-.246L.996 2.129a.843.843 0 0 1 0-1.23.843.843 0 0 1 1.23 0L6 4.644 9.746.898a.843.843 0 0 1 1.23 0 .843.843 0 0 1 0 1.23L6.603 6.505A.848.848 0 0 1 6 6.75Z"
                fill="#3E3232"
                fill-opacity="0.5"
              />
            </svg>
          </div>
          <button
            aria-label="box-button"
            class="w-12 h-12 bg-white-90 flex items-center justify-center rounded-xl w-12"
          >
            <svg
              fill="none"
              height="21"
              width="16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M13.625 0C14.641 0 15.5.86 15.5 1.875V18.75c0 .977-1.055 1.563-1.914 1.094L8 16.562l-5.625 3.282C1.515 20.312.5 19.727.5 18.75V1.875C.5.859 1.32 0 2.375 0h11.25Zm0 17.656V2.11c0-.117-.117-.234-.273-.234H2.57c-.117 0-.195.117-.195.234v15.547L8 14.375l5.625 3.281Z"
                fill="#3E3232"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
    <div
      class="container h-14 hidden my-5 sm:flex lg:hidden"
    >
      <label
        class="relative flex-1"
        for="search"
      >
        <div
          class="absolute left-6 top-4"
        >
          <svg
            data-testid="three-dots-icon"
            fill="none"
            height="18"
            width="6"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M3 13.063c1.172 0 2.188 1.015 2.188 2.187 0 1.21-1.016 2.188-2.188 2.188A2.184 2.184 0 0 1 .812 15.25c0-1.172.977-2.188 2.188-2.188Zm0-6.25c1.172 0 2.188 1.015 2.188 2.187 0 1.21-1.016 2.188-2.188 2.188A2.184 2.184 0 0 1 .812 9c0-1.172.977-2.188 2.188-2.188Zm0-1.875A2.184 2.184 0 0 1 .812 2.75C.813 1.578 1.79.562 3 .562c1.172 0 2.188 1.016 2.188 2.188 0 1.21-1.016 2.188-2.188 2.188Z"
              fill="#3E3232"
              fill-opacity="0.75"
            />
          </svg>
        </div>
        <div
          class="absolute right-4 top-3"
        >
          <svg
            data-testid="search-icon"
            fill="none"
            height="22"
            width="22"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M20.531 18.344a1.63 1.63 0 0 1 0 2.226 1.63 1.63 0 0 1-2.226 0l-4.649-4.687c-1.601 1.055-3.554 1.601-5.664 1.328-3.594-.508-6.484-3.438-6.953-6.992-.664-5.313 3.828-9.805 9.14-9.14 3.555.468 6.485 3.359 6.993 6.952a8.111 8.111 0 0 1-1.328 5.625l4.687 4.688ZM4.086 9.125c0 2.773 2.226 5 5 5 2.734 0 5-2.227 5-5 0-2.734-2.266-5-5-5-2.774 0-5 2.266-5 5Z"
              fill="#3E3232"
            />
          </svg>
        </div>
        <input
          class="w-full h-12 bg-white-90 placeholder-dark-100 rounded-xl outline-none px-14"
          id="search"
          name="search"
          placeholder="Search everything"
          type="text"
        />
      </label>
    </div>
  </header>
</DocumentFragment>
`;
